cekit/templates/template.jinja

```diff
@@ -19,20 +19,19 @@
 FROM {{ helper.base_image(from, version) }}
 
 # Environment variables
```

Comment:
Add this comment into the env section so it is not added when there are no env vars.